### PR TITLE
runit: add pages

### LINKED ISSUE #1808: page request: runit
`runit` is an alternative init system for various Unix-like OSes. It doesn't currently have `tldr` pages for its several commands. The programs in the `runit` suite include:

* `runsv` - The most important part, manages a single service/daemon.
* `runsvdir` - Starts and monitors multiple services using `runsv` from a directory.
* `runsvchdir` - Changes "runlevels."
* `sv` - Talks to `runsv` to control a service.
* `runit` - Acts as an init daemon: runs a startup script, starts `runsvdir` on the default "runlevel," responds to signals, and finally shuts down the machine.

There are also some ancillary tools, such as a logging service and a substitute `/sbin/init` that `exec`s the main `runit` program.

The main `runit` docs can be found [here](http://smarden.org/runit/).

### PR DESCRIPTION
**Pull request by alexbuzzbee**
This PR adds the following commands from [runit](http://smarden.org/runit/):

* runsv
* sv
* runsvdir
* runsvchdir
* runit

I haven't contributed here before, so I've probably done things wrong, but I've tried to follow the contributing guides as much as I can.

One of the commits in this PR is named wrong thanks to the GitHub web UI's inability to amend commits. It is "Update runsv.md" and should be "runsv: fix grammar"

Closes #1808.

----

- [x] The page (if new), does not already exist in the repo.

- [X] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [X] The page has 8 or fewer examples.

- [X] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited

- [X] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines

### CODE REVIEW

**Comment by alexbuzzbee** [comment]
I've fixed the example descriptions.

**Comment by agnivade** [comment]
Please move them under the "linux" folder as these commands are not available under windows.

**Comment by agnivade** [comment]
And we generally add just a single page in a single PR. It helps to review the changes better. Otherwise there are just lots of changes to track in multiple pages and github's review UI does not handle it well. It also increases the time taken to review a PR.

I am not suggesting to drop this PR and send a new one. Just something to keep in mind in future.

**Comment by sbrl** [comment]
Hey - thanks for the new pages! Looks like you're having some issues with the linter. We've got the style we prefer documented over [here](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md#markdown-format), and a few good examples are the pages for `tar`, `eyeD3`, and `ln`. If you need any assistance, just ask :smiley_cat:

**Comment by alexbuzzbee** [comment]
@agnivade There are a number of commands in "common" that are not available on Windows, and runit can be installed on macOS or SunOS; that said, I'll move it if you think that's better. I put all the pages in one PR because they are all part of the same package, but if that was incorrect I'll change it in any future PRs.

@sbrl I did read the guides (that's why it isn't much worse), I just forgot about the colons.

**Comment by agnivade** [comment]
Yes, there are lot of commands which are in common, which should not be. It is a mistake and should be fixed. What you need to do is create multiple copies of the same file and put it under linux, macos and sunos. (Yes it's ugly. Sorry, we don't have a better way yet).

> I put all the pages in one PR because they are all part of the same package

I get it. It's more from a PR reviewability perspective that I said this. Now that you will have to duplicate the pages in 3 folders, it will get even bigger.

>  I did read the guides (that's why it isn't much worse), I just forgot about the colons.

You don't need to learn the guide by heart. You can run the tests locally before pushing. 
```
npm install tldr-lint
tldrl {{page.md}}
```

**Comment by sbrl** [comment]
Ah, oops! I didn't think to check. That was a pre-written saved reply I've got for those who look like they are battling the tldr-bot.

**Comment by alexbuzzbee** [comment]
I've created copies of the pages in `linux/`, `osx/`, and `sunos/`, and removed the copies in `common/`.

File: pages/linux/runsvchdir.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# runsvchdir
+
+> Change the directory runsvdir uses by default.
```
Comment (agnivade) [inline pages/linux/runsvchdir.md]: Put runsvdir inside backticks(``)

File: pages/linux/runsv.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# runsv
+
+> Start and manage a runit service.
+
+- Start a runit service as yourself:
```
Comment (agnivade) [inline pages/linux/runsv.md]: "as the current user" might sound better.

File: pages/linux/runsvdir.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# runsvdir
+
+> Run an entire directory of services.
+
+- Start and manage all services in a directory as yourself:
```
Comment (agnivade) [inline pages/linux/runsvdir.md]: "as the current user" might sound better.

File: pages/linux/runsvdir.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
+`runsvdir {{path/to/services}}`
+
+- Start and manage all services in a directory as root:
+
+`sudo runsvdir {{path/to/services}}`
+
+- Start services in seperate sessions:
```
Comment (agnivade) [inline pages/linux/runsvdir.md]: typo at seperate

File: pages/linux/sv.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
+# sv
+
+> Control a running runsv service.
+
+- Start a service:
+
+`sv up {{path/to/service}}`
```
Comment (agnivade) [inline pages/linux/sv.md]: Doesn't this require root ?

Comment (alexbuzzbee) [reply pages/linux/sv.md]: Not necessarily, if the service runs as the user giving the command, but I'll add `sudo`.

File: pages/linux/runsvdir.md (hunk 4 above)
Comment (alexbuzzbee) [reply pages/linux/runsvdir.md]: Fixed

File: pages/linux/runsvdir.md (hunk 3 above)
Comment (alexbuzzbee) [reply pages/linux/runsvdir.md]: Fixed

File: pages/linux/runsvchdir.md (hunk 1 above)
Comment (alexbuzzbee) [reply pages/linux/runsvchdir.md]: Fixed

File: pages/linux/runsv.md (hunk 2 above)
Comment (alexbuzzbee) [reply pages/linux/runsv.md]: Fixed

**Pull request merged**